packages/app/src/specs/InlineSpecListHeader.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="h-64px items-center gap-8px mx-16px border-b-1 border-gray-900 grid grid-cols-[minmax(0,1fr),63px,24px]">
+    <div class="relative items-center group">
+      <div class="absolute inset-y-0 flex items-center pointer-events-none">
+        <MagnifyingGlassIcon
```

Comment:
Could you have used the `<i-cy-magnifying-glass_x24...` syntax here?
It makes looking for icons on the whole codebase that much easier.

Reply:
Converted all to `i-cy` style.